src/schema/submission.yaml

```diff
@@ -387,34 +389,41 @@ classes:
         required: false
         recommended: true
       description:
+        description: >-
+          A concise description about the Biospecimen source, the collection method, and the protocol which was followed to process the Biospecimen.
+        required: false
+      age_at_sampling:
+        required: true
+      vital_status_at_sampling:
         required: false
+        recommended: true
+      tissue:
+        description: >-
+          The tissue this Biospecimen originated from.
+        required: true
       isolation:
+        description: >-
+          Method or device employed for collecting/isolating the Sample.
         required: false
         recommended: true
       storage:
+        description: >-
+          Methods by which the Sample is stored (e.g. frozen in liquid nitrogen).
         required: false
         recommended: true
       individual:
         description: >-
           The Individual entity from which this Biospecimen was derived.
         inlined: false
         required: true
-      age_at_sampling:
-        required: true
-      vital_status_at_sampling:
-        required: false
-        recommended: true
-      tissue:
-        required: true
 
   Sample:
     mixins:
       - IdentifiedByAliasMixin
       - AttributeMixin
     description: >-
-      A sample is a limited quantity of something to be used for testing, analysis,
-      inspection, investigation, demonstration, or trial use. A sample is prepared from
-      a Biospecimen (isolate or tissue).
+      A sample is a limited quantity of something to be used for testing, analysis, inspection, investigation, demonstration, or trial use. 
+      A sample is prepared from a Biospecimen (isolate or tissue).
```

Comment:
(isolate or tissue) is not clear to me.

Reply:
Deleted the "isolate or tissue" part since we explain the Biospecimen. We might need to revisit the description though to see if the distinction between Biospecimen and Sample is clear enough.